.cirrus.yml

```diff
@@ -138,17 +138,19 @@ task:
     - libs
 
   debug_configure_script:
+    - brew update --force --quiet
+    - brew install llvm
```

Comment:
these should be part of an "install_script" which is neither debug nor release. the `install_script:` can come before the `debug_configure_script`.

Reply:
Fixed.